### PR TITLE
feat: add .gitignore; Error log show line of error and code now (#DVRL-81)

### PR DESCRIPTION
**Pull request by JohannesDienst-askui**

### CODE REVIEW

File: packages/askui-nodejs/src/execution/dsl.ts [hunk 1]
```diff
@@ -825,7 +411,7 @@ export class FluentFilters extends FluentBase {
    *     .customElement({
    *         customImage: './logo.png', // required
    *         name: 'myLogo', // optional
-   *         threshold: 0.5, // optional, defaults to 0.5
+   *         threshold: 0.9, // optional, defaults to 0.9
```
Comment (programminx-askui) [inline packages/askui-nodejs/src/execution/dsl.ts:414]: I think you've reverted the default parameter from @adi-wan-askui .

File: packages/askui-nodejs/src/execution/dsl.ts [hunk 2]
```diff
@@ -1935,7 +1552,7 @@ export class FluentFiltersCondition extends FluentBase {
...
```
Comment (programminx-askui) [inline packages/askui-nodejs/src/execution/dsl.ts:1555]: Same here.

**Review by programminx-askui — CHANGES_REQUESTED** [review]
@mlikasam-askui  has another [PR](https://github.com/askui/askui/pull/761) open, which change the DSL. I we should be carefull regarding merging.

File: packages/askui-nodejs/example_projects_templates/templates/askui-helper-windows.nj [hunk 3]
```diff
@@ -17,12 +17,16 @@ beforeAll(async () => {
 });
 
 beforeEach(async () => {
+  /* Uncomment to enable video recording
   await aui.startVideoRecording();
+  */
```
Comment (mlikasam-askui) [inline packages/askui-nodejs/example_projects_templates/templates/askui-helper-windows.nj:22]: What do you think about adding a param to the askui init called `--with-video-recording` and set it to false as a default a value?

Comment (programminx-askui) [reply packages/askui-nodejs/example_projects_templates/templates/askui-helper-windows.nj:22]: I'm removing video recording for each customer. And nobody has complaint about it. 

The idea was to uncomment it, so the user has the possibility to enable it later.

Comment (programminx-askui) [reply packages/askui-nodejs/example_projects_templates/templates/askui-helper-windows.nj:22]: And for replacing the Lagacy Controller Video recording needs to be removed.

File: packages/askui-nodejs/src/execution/dsl.ts (hunk 1 above)
Comment (JohannesDienst-askui) [reply packages/askui-nodejs/src/execution/dsl.ts:414]: Yes, indeed.

File: packages/askui-nodejs/example_projects_templates/templates/askui-helper-windows.nj (hunk 3 above)
Comment (JohannesDienst-askui) [reply packages/askui-nodejs/example_projects_templates/templates/askui-helper-windows.nj:22]: Do you think we should merge this or should I add a flag? For me both are fine 😉 .

Comment (programminx-askui) [reply packages/askui-nodejs/example_projects_templates/templates/askui-helper-windows.nj:22]: Let's merge it.

**Review by programminx-askui — APPROVED** [review]

**Pull request merged**